pages/users/[id].tsx

```diff
@@ -61,6 +76,11 @@ export default function User({ loginContext }: Props) {
     API.MetricsConfigResponse | undefined
   >(undefined)
   const [$fetched, $setFetched] = useState(false)
+  useEffect(() => {
+    if (typeof window !== 'undefined') {
+      $setDomain(`https://` + window.location.hostname)
+    }
+  }, [])
```

Comment:
Maybe this is okay to inline outside of the `useEffect` hook? It could also just be set as a constant as well.

Reply:
Yeah, fair call. I elected to use a static string now. I wanted initially to verify how it works.